pkg/kube/kube_handler.go

```diff
@@ -143,7 +144,7 @@ type userHandler struct {
 	// The namespace in which (currently) app repositories are created.
 	kubeappsNamespace string
 
-	// clientset using the pod serviceaccount for the default cluster
+	// clientset using the pod serviceaccount for the specific cluster
 	svcClientset combinedClientsetInterface
```

Comment:
if you are renaming this to kubeappsSVCClientset, why don't you change this one as well? (It's the same thing)

Reply:
I didn't think it was the same thing. The one I've renamed to `kubeappsSvcClientSet` uses the service account on which kubeapps is installed, always, where as this `svcClientset` is for the current cluster in a multicluster env (think requesting namespaces to populate the selector). Yes, it will be the same thing in a single cluster environment but isn't generally.